### DISCUSSION TITLE
Copilot on Mac

### DISCUSSION THREAD

**Discussion by IgorRudy** [community/community#15737] [Copilot Conversations]
Hi all, 

i'm pretty new on programming. 
Copilot working great for me with VS Code on mac, 
but i dont find a way to install it on the VS for mac.
Google doesnt help alot. 
Is there a way to install Copilot on the VS for mac?

Thanks in advance!

I

**Comment by Zure1** [comment]
Same issue here. 
I think the problem is that I can't find a way to link my GitHub account to VS4M. I can only login with my Microsoft Account when trying to add an Account.

@IgorRudy  Did you manage to find a way to install GitHub Copilot in VS4M?

Is it even possible to install Copilot to VS4M?

**Comment by LightningBit** [comment]
Note: GitHub Copilot is not currently available for use with Visual Studio for Mac.

https://docs.github.com/en/copilot/getting-started-with-github-copilot/getting-started-with-github-copilot-in-visual-studio
